jupyterhub/templates/scheduling/user-placeholder/statefulset.yaml

```diff
@@ -0,0 +1,54 @@
+
+{{- /* 
+These user-dummy pods can be used to test cluster autoscaling in a
+controlled fashion.
+
+Example:
+$ echo 'Simulating four users...'
+$ kubectl patch deployment user-dummy --patch '{"spec": {"replicas": 4}}'
+*/}}
+{{- if .Values.scheduling.userPlaceholder.enabled -}}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: user-placeholder
+  labels:
+    {{- include "jupyterhub.labels" . | nindent 4 }}
+spec:
+  podManagementPolicy: Parallel
+  replicas: {{ .Values.scheduling.userPlaceholder.replicas }}
+  selector:
+    matchLabels:
+      {{- include "jupyterhub.matchLabels" . | nindent 6 }}
+  serviceName: "user-placeholder"
+  template:
+    metadata:
+      labels:
+        {{- /* Changes here will cause the Deployment to restart the pods. */}}
+        {{- include "jupyterhub.matchLabels" . | nindent 8 }}
+        hub.jupyter.org/pod-kind: "user"
+    spec:
+      {{- if .Values.scheduling.podPriority.enabled }}
+      priorityClassName: jupyterhub-user-placeholder-priority
+      {{- end }}
+      {{- if .Values.scheduling.userScheduler.enabled }}
+      schedulerName: user-scheduler
+      {{- end }}
+      {{- $_ := merge (dict "podKind" "user") . }}
+      {{- $dummy := include "jupyterhub.prepareScope" $_ }}
```

Comment:
This line uses a helper to set information in the $_ variable, this relevant information about the users affinity info for example, can be extracted thanks to the information is provided to the config.yaml but within extraConfig but instead being provided in a explicit fields @minrk .